src/pretix/control/forms/discounts.py

```diff
@@ -0,0 +1,99 @@
+#
+# This file is part of pretix (Community Edition).
+#
+# Copyright (C) 2014-2020 Raphael Michel and contributors
+# Copyright (C) 2020-2021 rami.io GmbH and contributors
+#
+# This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General
+# Public License as published by the Free Software Foundation in version 3 of the License.
+#
+# ADDITIONAL TERMS APPLY: Pursuant to Section 7 of the GNU Affero General Public License, additional terms are
+# applicable granting you additional permissions and placing additional restrictions on your usage of this software.
+# Please refer to the pretix LICENSE file to obtain the full terms applicable to this work. If you did not receive
+# this file, see <https://pretix.eu/about/en/license>.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied
+# warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero General Public License for more
+# details.
+#
+# You should have received a copy of the GNU Affero General Public License along with this program.  If not, see
+# <https://www.gnu.org/licenses/>.
+#
+from decimal import Decimal
+
+from django import forms
+from django.utils.translation import gettext_lazy as _
+
+from pretix.base.channels import get_all_sales_channels
+from pretix.base.forms import I18nModelForm
+from pretix.base.forms.widgets import SplitDateTimePickerWidget
+from pretix.base.models import Discount
+from pretix.control.forms import ItemMultipleChoiceField, SplitDateTimeField
+
+
+class DiscountForm(I18nModelForm):
+    class Meta:
+        model = Discount
+        localized_fields = '__all__'
+        fields = [
+            'active',
+            'internal_name',
+            'sales_channels',
+            'available_from',
+            'available_until',
+            'subevent_mode',
+            'condition_all_products',
+            'condition_limit_products',
+            'condition_min_count',
+            'condition_min_value',
+            'condition_apply_to_addons',
+            'benefit_discount_matching_percent',
```

Comment:
Eyecandy: Add a `attrs={'addon_after': _('%')}` to the widget

Reply:
done